doc/contribute/contribute_non-apache.rst

```diff
@@ -0,0 +1,70 @@
+.. _contribute_non-Apache:
+
+Contributing non-Apache 2.0 licensed components
+###############################################
+
+Importing code into the Zephyr OS from other projects that use a license
+other than the Apache 2.0 license needs to be fully understood in
+context and approved by the `Zephyr governing board`_.
+
+.. _Zephyr governing board:
+   https://www.zephyrproject.org/about/organization
+
+By carefully reviewing potential contributions and also enforcing a
+:ref:`DCO` for contributed code, we insure that
+the Zephyr community can develop products with the Zephyr Project
+without concerns over patent or copyright issues.
+
+Submission and review process
+*****************************
+
+All contributions to the Zephyr project are submitted through GitHub
+pull requests (PR) following the Zephyr Project's :ref:`Contribution workflow`.
+For contributions using a using a license other than Apache 2.0 these
+additional steps must be followed:
+
+#. Complete a README for your code component and add it to your source
+   code PR.  A recommended README template can be found in
+   :file:`doc/contributing/code_component_README` (and included
+   `below`_ for reference)
+
+#. Email a copy of this README to the Zephyr Technical
```

Comment:
no need to send email if it is part of the pull request, so I would remove this.

Reply:
I got that from the PPT presentation.  I'll remove it here.